sceptre/scipool/config/develop/sc-tag-options.yaml

```diff
@@ -12,7 +12,8 @@ dependencies:
 sceptre_user_data:
   Departments: !file sc-tag-options/internal/Departments.json
   Projects: !file sc-tag-options/internal/Projects.json
-  CostCenters: !file https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.3.6/tags/SageFinancialProgramCodes.json
+  CostCenters: !from_json
+    - !request 'https://finops-api.sageit.org/tags?limit=25'
```

Comment:
This should be `mips-api.finops.sageit.org`, and this change will also remove the "other" option since I removed it from the lambda with an intention of adding it to `sc-tag-options.j2` in #808. Let's hold off on this change until I split "other" and "no program" in the lambda configuration to address your point on that PR.

Reply:
url has been fixed